### DISCUSSION TITLE
Consulta

### DISCUSSION THREAD

**Discussion by bsantillan** [frlp-utn-ingsoft/catedra#19] [Q&A] [answered]
Buenas tardes, estoy teniendo un problema al iniciar el servidor e ingresar a la sección localhost:8000/admin.
No me aparece la pantalla de inicio del administrador, la cual debería ser la siguiente:
![Pantalla de logeo de admin](https://github.com/frlp-utn-ingsoft/catedra/assets/92806116/def6e496-1e0c-42b5-a950-52665a1ad40c)

Y en mi ejecución me aparece lo siguiente:
![Lo que me salta](https://github.com/frlp-utn-ingsoft/catedra/assets/92806116/8d20b5f5-2f55-41d4-a461-1638516574a3)

Estoy hace un rato largo viendo que puede ser y no encuentro solución.

Por si necesitan leer el codigo les dejo el link a mi repositorio:
https://github.com/bsantillan/Ingenieria-y-Calidad-de-Software

Saludos

**Comment by RodrigoJacznik** [comment, marked as answer]
Buenas, fijate que en este commit: https://github.com/bsantillan/Ingenieria-y-Calidad-de-Software/commit/dcad0a8fb34c2406f549602eb057b935c84d64a7

Cambiaste la variable TIME_ZONE y le pusiste un valor no valido. Volvele a poner UTC.

**Reply by bsantillan** [reply]
Era ese el error.

Muchas gracias.
